BlueShift-iOS-SDK/InApps/BlueShiftInAppNotification.m

```diff
@@ -441,7 +441,7 @@ - (instancetype)initFromEntity: (InAppNotificationEntity *) appEntity {
                     
                     self.position = kInAppNotificationModalPositionCenterKey;
                     self.dimensionType = kInAppNotificationModalResolutionPercntageKey;
-                    self.width = 100;
+                    self.width = kInAppNotificationDefaultWidth;
                     self.height = 100;
```

Comment:
keep the value of height to constants, like  width

Reply:
+1

Reply:
fixed.